common/utils/convert-image-uri.ts

```diff
@@ -100,15 +100,17 @@ function prismicTemplateParts( // gets the params from the original Prismic imag
 
 export function convertIiifImageUri(
   originalUri: string,
-  requiredSize: number | 'full'
+  requiredSize: number | 'full',
+  sizeByHeight?: boolean
 ): string {
   if (determineIfGif(originalUri) || !originalUri.startsWith(iiifImageUri)) {
     return originalUri;
   } else {
     const imageIdentifier = originalUri.split(iiifImageUri)[1].split('/', 2)[0];
 
+    const size = sizeByHeight ? `,${requiredSize}` : `${requiredSize},`;
```

Comment:
Could you explain the difference? (I see the `,` is placed elsewhere but what does it do?

Reply:
If the comma is after the value it requests and image by width, if it's before it requests it by height.

See: https://iiif.io/api/image/3.0/#42-size

Reply:
funky! thanks :)